README.md

```diff
@@ -75,10 +66,7 @@ Possible variants of the last step include
 ```shell
 pip install -e .[dev,test]                 # optionally install dev and test dependencies
 CMAKE_ARGS=-DCUDA_ARCH=86 pip install -e . # override CUDA_ARCH
-SKIP_CUSTOM_OPS=1 pip install -e .         # skip building CUDA extension (non-Linux)
-
-# use parallel CMake build with `nproc` threads
-CMAKE_BUILD_PARALLEL_LEVEL=$(nproc) pip install -e .
+SKIP_CUSTOM_OPS=1 pip install -e .         # skip building CUDA extension
```

Comment:
nit: maybe add "(no effect on non-Linux)" to comment?

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/944/commits/7d372f6d16783b07cb916a1f350e6efb4a3b32a3